### PR TITLE
Audio: Multiband-DRC: Move blob configuration scripts to module directory

### PR DESCRIPTION
**Pull request by singalsu**

### CODE REVIEW

File: src/audio/multiband_drc/tune/multiband_drc_paths.m [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	common = '../../../../tools/tune/common';
+	crossover = '../../../../tools/tune/crossover';
+	drc = '../../../../tools/tune/drc';
+	eq = '../../../../tools/tune/eq';
+	# After #9215 merge use this:
+	# crossover = '../../crossover/tune';
+	# After #9188 merge use this:
+	# drc = '../../drc/tune';
+	# After #9187 merge use this:
+	# eq = '../../eq_iir/tune';
```
Comment (lgirdwood) [inline src/audio/multiband_drc/tune/multiband_drc_paths.m:21]: Btw, does Matlab/Octave support PATHs via environment variable ? If so, might be easiest to have a script that sets these for the (like Zephyr has convenience script to setup build environment)

**Review by lgirdwood — APPROVED** [review]

File: src/audio/multiband_drc/tune/multiband_drc_paths.m (hunk 1 above)
Comment (singalsu) [reply src/audio/multiband_drc/tune/multiband_drc_paths.m:21]: Yes, the environment variables can be supported:

```
octave:1> getenv('SOF_WORKSPACE')
ans = /home/singalsu/work/current/sof
```

Would above be better e.g. as `tune = sprintf('%s/sof/tools/tune', getenv('SOF_WORKSPACE'));` ?

Comment (lgirdwood) [reply src/audio/multiband_drc/tune/multiband_drc_paths.m:21]: Btw, lets track this as a long term feature - ideally long term west would enable this when we configure the environment.

**Comment by kv2019i** [comment]
Release reminder - one week to v2.11-rc1.

**Pull request merged**